guides/relationships/features/links-mode.md

```diff
@@ -0,0 +1,199 @@
+# LinksMode
+
+- ⮐ [Relationships Guide](../index.md)
+
+---
+
+**LinksMode** is a special feature that can be activated for any `belongsTo` or `hasMany` relationship.
+
+It allows that relationship to be fetched using the standard `request` experience instead of via the legacy `adapter` interface.
+
+LinksMode behaves *slightly* differently depending on whether
+you are using Model (including via [Legacy Mode](../../reactive-data/legacy/overview.md)) or [Polaris Mode](../../reactive-data/polaris/overview.md). We'll explain this nuance below.
+
+> [!TIP]
+> The next-generation of reactive data which replaces Model is SchemaRecord.
+> SchemaRecord has two modes, Legacy - which emulates all of Model's
+> behaviors and APIs, and Polaris - a new experience which we intend
+> to make default in Version 6.
+
+## How Does It Work?
+
+### Related Link Becomes Required
+
+Relationships in WarpDrive are stored using the same top-level structure as resource documents, a structure
+adopted from the [JSON:API](https://jsonapi.org) specification.
+
+- **meta** *(optional)*: an object of arbitrary extra information about the relationship
+- **links** *(optional)*: an object containing various links for fetching and managing the relationship
+- **data**: the membership (state of) the relationship
+
+This is roughly described by the interface below:
+
+```ts
+interface Relationship {
+  meta?: Record<string, Value>;
+  links?: Links;
+}
+
+interface ResourceRelationship extends Relationship {
+  data: { type: string; id: string | null; lid: string } | null;
+}
+
+interface CollectionRelationship extends Relationship {
+  data: { type: string; id: string | null; lid: string }[]
+}
+```
+
+When `linksMode` is activated for a relationship, it is required that a related link is present.
+
+```ts
+interface LinksModeRelationship {
+  meta?: Record<string, Value>;
+
+  // no longer optional
+  links: {
+    related: string | { href: string };
+
+    // other links still as desired
+  }
+}
+```
+
+### Related Links May be provided by Handlers
+
+This means that in order to use links mode a relationship payload given to the cache MUST contain this related link. If your API does not provide this link, a request handler could be utilized to decorate an API response to add them provided that your handlers (or your API) are able to understand that link.
```

Comment:
```suggestion
This means that, in order to use links mode, a relationship payload given to the cache MUST contain this related link. If your API does not provide this link, a request handler could be utilized to decorate an API response to add them provided that your handlers (or your API) are able to understand that link.
```

Maybe link "request handler" to https://api.emberjs.com/ember-data/release/classes/%3CInterface%3E%20Handler ?